### PR TITLE
[community/kep pages] Add automated handling for links from imported files

### PR DESCRIPTION
**Pull request by abiogenesis-now**
@chenopis PTAL

I've tried to generalize this solution and not make it too brittle---basically if you mark the import config file (e.g. `community.yml`) with `gen-absolute-links`, it processes URLs from the imported pages as follows, depending on their type:
* `#id-on-page` - leaves these URLs as they are
* `/path/url` - adds the remote repo URL prefix
* `path/url`, `./path.url` - adds the remote repo URL and the relevant file's subpath as a prefix
* `http://`, `https://` - leaves these URLs as they are
* `mailto:` - leaves these URLs as they are

Regex works for now, but people well-versed in parsing can likely come up with a cleaner solution.

Some of the URLs are broken because things in community have been moved around, I'll follow up on the CI report and fix them accordingly.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6926)

### CODE REVIEW

**Comment by abiogenesis-now** [comment]
Broken links fixed with https://github.com/kubernetes/community/pull/1587

File: update-imported-docs/update-imported-docs.go [hunk 1]
```diff
@@ -159,25 +176,33 @@ func main() {
   fmt.Fprintf(os.Stdout, "\n\t\t\t*\t*\t*\n\nDocs imported! Run 'git add .' 'git commit -m <comment>' and 'git push' to upload them.\n")
 }
 
-func copyFile(src, dst string) error {
```
Comment (abiogenesis-now) [inline update-imported-docs/update-imported-docs.go]: removing this function, i checked and it's not being used anywhere

**Comment by chenopis** [comment]
I'm going to merge this and do more extensive testing w/ the kubectl docs.

**Pull request merged**